pkg/internal/sqlsmith/schema.go

```diff
@@ -359,25 +392,10 @@ func (s *Smither) extractIndexes(
 					Direction: dir,
 				})
 			}
-			row := s.db.QueryRow(fmt.Sprintf(`
```

Comment:
we don't we need this 19.2 fallback logic any more?

Reply:
I don't think so - each roachtest is run on its branch now, I thought.

Reply:
ah yes, that was just changed in the beginning of november. so this lgtm